page-changelog.php

```diff
@@ -3,6 +3,80 @@
+<li>Hide strength indication after password change and hide notifications after time-out or on delete</li>
+<li>Fixes to inconsistent language and translation support, show language code for unknown languages</li>
+<li>Fix icons for share/public folders, fix spinner positions in share tab</li>
+<li>Close user menu when clicking on other menus</li>
+<li>Sidebar fixes and hiding sidebar for trashbin view</li>
+<li>Move alt text for favorite action to image</li>
+<li>Sidebar should not open, when renaming a file on mobile</li>
+<li>Do not register sidebar panels when no sidebar</li>
+<li>Update process: state which step we are going to start and warn if it might be slow</li>
+</ul>
+<li>User Management and LDAP fixes:</li>
+<ul>
+<li>LDAP fixes for quota, user mapper, initial user creation and connectivity</li>
+<li>Fix group admin settings, group assignment when group name is a number</li>
+<li>Enable proper CardDAV cookie authentication</li>
+</ul>
+<li>Documentation, reliability and stability fixes</li>
+<ul>
+<li>Expose syslog tag in the configuration</li>
+<li>Update list of deprecated methods (documentation)</li>
+<li>Fixes to text editor</li>
+<li>Fixes to activity app</li>
+<li>Create several repair steps in update process to clean up shares</li>
+<li>Update certificate bundle</li>
+<li>Fix multiple issues for IE 8 and 9</li>
+<li>Memcache warning with memcached</li>
+<li>Fix File versioning with encryption</li>
+<li>Update the unencrypted size for versions</li>
+<li>Fix mtime PROPPATCH to be "lastmodified" instead of "getlastmodified"</li>
+<li>Make sure that remote shares use the correct uid casing</li>
+<li>Variables don't have a class, so we can't use toString() on it</li>
+<li>Use IRequest's `getScriptName` functionality instead of $_SERVER</li>
+<li>Don't trigger the scroll event of every single item we filter in the file list</li>
+</ul>
+</ul>
+Download: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2">owncloud-8.2.1.tar.bz2</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip">owncloud-8.2.1.zip</a></br>
+MD5: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.md5">owncloud-8.2.1.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.md5">owncloud-8.2.1.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.sha256">owncloud-8.2.1.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.sha256">owncloud-8.2.1.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.asc">owncloud-8.2.1.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.asc">owncloud-8.2.1.zip.asc</a></br>
+Packages: <a href="http://software.opensuse.org/download/package?project=isv:ownCloud:community:8.1&package=owncloud">on the Open Build Service</a></br>
```

Comment:
Should be https://download.owncloud.org/download/repositories/stable/owncloud/ or https://download.owncloud.org/download/repositories/8.2/owncloud/